src/card.css

```diff
@@ -0,0 +1,77 @@
+.card {
+  width: 200px;
+  background-color: var(--color-white);
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__product {
+  display: flex;
+  min-height: 134px;
```

Comment:
you don't need it, the photo dictates container's height
```suggestion
```

Reply:
And here the photo dictates the height as you said but the height was 133 px, but in figma it was 134 and the tests failed because of that, that is why I set the min-height to 134px

Reply:
Okay